src/Data/Interval.hs

```diff
@@ -745,10 +746,14 @@ instance (RealFrac r, Floating r) => Floating (Interval r) where
       (PosInf, _) -> whole
       (Finite ub, in2)
         | ub - lb > pi -> whole
-        | ub - lb == pi && in1 == Open && in2 == Open && modpi lb /= -1/2 -> throw LossOfPrecision
+        -- the next case corresponds to (-inf, tan lb) + (tan lb, +inf),
```

Comment:
Fixed

Reply:
Actually, it's correct! Since `ub - lb == pi` we have `tan lb == tan ub`. It's just that both ends are open, so the result is the whole real axis without one point.